trace/nox.py

```diff
@@ -28,7 +28,8 @@ def unit_tests(session, python_version):
     # Install all test dependencies, then install this package in-place.
     session.install('mock', 'pytest', 'pytest-cov',
                     'django', 'flask', 'webapp2', 'webob',
-                    'google-cloud-trace', 'mysql-connector==2.1.6')
+                    'google-cloud-trace', 'mysql-connector==2.1.6',
```

Comment:
Can we move these into a `requirements-test.txt` file?

Reply:
Done.